vcpkg/users/buildsystems/msbuild-integration.md

```diff
@@ -1,23 +1,38 @@
 ---
-title: MSBuild Integration
+title: vcpkg in MSBuild projects
 description: Integrate vcpkg into an MSBuild or Visual Studio project.
-ms.date: 11/30/2022
+author: vicroms
+ms.author: viromer
+ms.prod: vcpkg
+ms.date: 9/18/2023
 ---
-# MSBuild integration (Visual Studio)
+# vcpkg in MSBuild projects
 
 ## Integration methods
 
 ### User-wide integration
 
-```console
+To use vcpkg in your MSBuild projects run the following command:
+
+```Console
 vcpkg integrate install
 ```
 
-This will implicitly add Include Directories, Link Directories, and Link Libraries for all packages installed with vcpkg to all VS2015, VS2017, VS2019, and VS2022 MSBuild projects. We also add a post-build action for executable projects that will analyze and copy any DLLs you need to the output folder, enabling a seamless F5 experience.
+You only need to run the [`vcpkg integrate install`](../../commands/integrate.md#vcpkg-integrate-install)
```

Comment:
Maybe talk about how this works if you have multiple copies of vcpkg on your system (this makes this copy of vcpkg the one MSBuild recognizes).